domenetjenester/iay/src/main/java/no/nav/foreldrepenger/abakus/domene/iay/YtelseEntitet.java

```diff
@@ -252,7 +252,7 @@ public boolean equals(Object o) {
 
     @Override
     public int hashCode() {
-        return Objects.hash(relatertYtelseType, periode, saksnummer);
+        return Objects.hash(relatertYtelseType, temaUnderkategori, periode, saksnummer);
```

Comment:
Må i såfall i equals også

Reply:
Ligger allerede i equals

Reply:
Her bør dere tenke dere om. Disse er designet for å håndtere både infotrygd, arena og vl

Reply:
Temaunderkategori fins ikke for Arena/VL . Husker ikke om den er NULL eller '-'.
Bør testes

Reply:
Men man bør vel kunne diffe ytelser som har ulik temaunderkategori uansett?

Reply:
OK. Denne forklarer sikkert en del spennende avvik. Bra oppdagelse

Reply:
Creds til @olejri